jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2924,9 +2924,10 @@ public String getTypeName() {
  * 		Repeatable annotations are annotated with Repeatable annotation which tells the 
  * 		container annotation for this repeatable annotation.
  * 		Example = 
- * 
+ * 		<code>
  * 		@interface ContainerAnnotation {RepeatableAnn[] value();}
  * 		@Repeatable(ContainerAnnotation.class)
+ * 		</code>
```

Comment:
This documentation needs more formatting, as is, it is presented as a single paragraph:
> Terms used for annotations : Repeatable Annotation : An annotation which can be used more than once for the same class declaration. Repeatable annotations are annotated with Repeatable annotation which tells the container annotation for this repeatable annotation. Example =

It need not be addressed in this pull request.

Using `<code>...</code>` doesn't seem to work, but this does:
```
 * 			<pre>{@code
 * 		 	@interface ContainerAnnotation {RepeatableAnn[] value();}
 * 			@Repeatable(ContainerAnnotation.class)
 * 			}</pre>
```

Reply:
Correction: we want `<pre>` and `</pre`> but we can't use `{@code` where the closing `}` is not on the same line without breaking earlier versions. This should be:
```
 * 		<pre><code>
 *	 	{@literal @}interface ContainerAnnotation {RepeatableAnn[] value();}
 *		{@literal @}Repeatable(ContainerAnnotation.class)
 *		</code></pre>
```